packages/apps/hr/src/features/expenseSlice.js

```diff
@@ -272,6 +309,13 @@ const expenseSlice = createSlice({
         manageTotal: true,
       },
     );
+    builder.addCase(silenceSearchExpense.pending, state => {
+      state.loadingMyExpense = false;
+    });
```

Comment:
remove this please, the objective is to have a silent refresh so we must not touch on loading attributes